Ch03/operation.cpp

```diff
@@ -0,0 +1,24 @@
+#include "../std_lib_facilities.h"
+
+int main() {
+	string operation;
+	double opd1;
+	double opd2;
```

Comment:
What if I put string in opd1 or opd2? Can we accept it as string and then convert to double using stod(opd1)?